src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -114,6 +116,25 @@ protected static void ignorePossibleException() {
         assertTrue(true);
     }
     
+    protected static String getPopulatedErrorMessage(String messageTemplate, String userInput,
+                                                     String fieldName, String errorReason) {
+        return getPopulatedErrorMessage(messageTemplate, userInput, fieldName, errorReason, 0);
+    }
+
+    protected static String getPopulatedErrorMessage(String messageTemplate, String userInput,
+                                                     String fieldName, String errorReason, int maxLength) {
+        try {
+            Method getPopulatedErrorMessage =
+                    FieldValidator.class.getDeclaredMethod("getPopulatedErrorMessage", String.class,
+                            String.class, String.class, String.class, int.class);
+            getPopulatedErrorMessage.setAccessible(true);
+            return (String) getPopulatedErrorMessage.invoke(null, messageTemplate, userInput, fieldName,
+                                                            errorReason, maxLength);
```

Comment:
@wkurniawan07  Yup yup, it is perfectly valid if this helper is provided in the `BaseTestCase`. I totally misread your previous comment and was under the impression that you suggested using reflection directly in each test =(